pipelinewise/cli/pipelinewise.py

```diff
@@ -1448,11 +1482,14 @@ def sync_tables(self):
             self.logger.exception(exc)
             self.send_alert(message=f'Failed to sync tables in {tap_id} tap', exc=exc)
             sys.exit(1)
+        except PreRunChecksException:
+            sys.exit(1)
         except Exception as exc:
             self.send_alert(message=f'Failed to sync tables in {tap_id} tap', exc=exc)
             raise exc
         finally:
-            utils.silentremove(cons_target_config)
+            if cons_target_config:
```

Comment:
I don't think this is needed.
`utils.silentremove` function would not error if the the given file doesn't exist, hence the `silent` in the name

Reply:
It will, if its input parameter be None. there are some cases that checks can be failed before defining value for `cons_target_config`